.github/workflows/sync-strings.yml

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+name: "Sync Strings"
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  main:
+    name: "Sync Strings"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Discover Focus Beta Version"
+        id: focus-android-beta-version
+        uses: mozilla-mobile/focus-android-beta-version@2.0.0
```

Comment:
This is the only thing that seems to be missing. The script hardcodes the Fenix project so it cannot be used directly.
Looked into adding a new repo similar to https://github.com/mozilla-mobile/fenix-beta-version for Focus also but going through that code it seems we can adapt that project to be a general one that can be used for many other Github actions from Mozilla or others and opened a new ticket for that - https://github.com/mozilla-mobile/fenix-beta-version/issues/2.

Reply:
Let's use https://github.com/mozilla-mobile/fenix-beta-version/releases/tag/3.0.0 .